chucks-pick3-web/angular-app/src/app/components/home.component/home.component.html

```diff
@@ -20,18 +20,19 @@
   <div class="col-md-4 middlebdr no-pad">
     <a [routerLink]="['/demo']" [queryParams]="{showSubNavBar: false}">
       <div class="grediant">
-        <img src="assets/images/screens-01.png">
+        <img src="src/app/assets/images/screens-01.png">
       </div>
-      <div class="greenpatch ">
-        <p  style="padding:0px 7px; margin: 0px;">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec feugiat tristique semper. Nullam lacinia
-        tincidunt lectus vel volutpat.</p>
+      <div class="greenpatch">
+        <p style="padding:0px 7px; margin: 0px;">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec feugiat
+          tristique semper. Nullam lacinia
+          tincidunt lectus vel volutpat.</p>
       </div>
     </a>
   </div>
   <div class="col-md-4 no-pad-left">
     <a [routerLink]="['/sign-up']" [queryParams]="{showSubNavBar: false}">
       <div class="grediant">
-        <img src="/assets/images/screens_Artboard 1-03.png" class="signup" alt="Sign-Up" />
+        <img src="src/app/assets/images/screens_Artboard 1-03.png" class="signup" alt="Sign-Up"/>
```

Comment:
Not Sure about this change.  Does this still work?  I think it is okay to start paths to images and css files with just `assets/images` and `assets/css`;  Please revert back.

Reply:
reverted to assets/images